d3ploy/supply_driven_deployment_inst.py

```diff
@@ -98,6 +98,14 @@ class SupplyDrivenDeploymentInst(Institution):
         uilabel="Record to Text",
         default=False)
 
+    installed_cap = ts.Bool(
+        doc="Indicates whether or not to use installed capacity as the" +
```

Comment:
why does this exist in so many places... can this function be inherited from the base class instead?